src/helpers/WalletLanguage.ts

```diff
@@ -0,0 +1,19 @@
+import * as english from './languages/english.json';
+import * as spanish from './languages/spanish.json';
+
+export default class WalletLanguage {
+    getLanguage() {
+        const lang: any = ((navigator.languages && navigator.languages[0] && navigator.language) || 'en').substr(0, 2);
+        return lang;
+    }
+    getText() {
+        const moduleLanguage: any = { es: spanish, en: english };
```

Comment:
this doesn't look dynamic, if someone adds a new language they'd need to come here and modify this file, no?